### PR TITLE
avatar done

### PR DESCRIPTION
**Pull request by ashifkhn**

### CODE REVIEW

**Review by Pratham82 — COMMENTED** [review]
- Mention the avatar size classes in the description.
- Align code in the center
- Add left padding to the sidebar
- Vertically align the content in the navbar using `align-items`

File: components/Badge/badge.html [hunk 1]
```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
```
Comment (sakshi006) [inline components/Badge/badge.html:2]: You can add some styling to the badges as well. Try to make them cover a part of the image. Add different background to highlight a badge.

**Review by sakshi006 — COMMENTED** [review]
Requested changes

File: components/Badge/styleBadge.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    height: 3rem;
+}
+
+.badge_tiny{
+    color: rgb(0, 0, 0);
+    position: relative;
+    top: -1rem;
+    left: -0.5rem;
+    height: 8px;
+    padding: 7px;
```
Comment (sakshi006) [inline components/Badge/styleBadge.css:24]: ` padding: 3px 7px 12px 7px;`

File: components/Badge/styleBadge.css [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+.avatar_tiny_container{
+    display: flex;
+    width: 3rem;
+    height: 3rem;
+}
+
+.badge_tiny{
+    color: rgb(0, 0, 0);
+    position: relative;
+    top: -1rem;
```
Comment (sakshi006) [inline components/Badge/styleBadge.css:21]: ` top: -0.4rem;
    left: -0.7rem;
`

File: components/Badge/styleBadge.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    font-size: 1rem;
+    font-weight: bolder;
+    background-color: rgb(255, 0, 0);
+    border-radius: 20px;
+}
+
+.badge_small{
+    color: rgb(0, 0, 0);
+    position: relative;
+    top: -4rem;
```
Comment (sakshi006) [inline components/Badge/styleBadge.css:34]: `
top: -3.5rem;
    right: 1.5rem;
`

File: components/Badge/styleBadge.css [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    right: 1rem;
+    height: 8px;
+    padding: 7px;
```
Comment (sakshi006) [inline components/Badge/styleBadge.css:37]: ` padding: 2px 8px 2px 8px `

File: components/Badge/styleBadge.css [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    font-size: 1rem;
+    font-weight: bolder;
+    background-color: rgb(255, 0, 0);
+    border-radius: 20px;
+}
+
+.badge_large{
+    color: rgb(0, 0, 0);
+    position: relative;
+    top: -6rem;
```
Comment (sakshi006) [inline components/Badge/styleBadge.css:61]: `top: -5.5rem;
    right: 2rem;`

File: components/Badge/styleBadge.css [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+    right: 1rem;
+    padding: 7px;
```
Comment (sakshi006) [inline components/Badge/styleBadge.css:63]: `    padding: 7px 14px 7px 12px`

File: components/Badge/styleBadge.css [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+}
+
+
+.badge_medium{
+    color: rgb(0, 0, 0);
+    position: relative;
+    top: -5rem;
+    right: 1rem;
+    height: 8px;
+    padding: 7px;
```
Comment (sakshi006) [inline components/Badge/styleBadge.css:51]: ` padding: 2px 8px 2px 8px; `

File: components/Badge/styleBadge.css [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+    font-weight: bolder;
+    background-color: rgb(255, 0, 0);
+    border-radius: 20px;
+}
+
+
+.badge_medium{
+    color: rgb(0, 0, 0);
+    position: relative;
+    top: -5rem;
```
Comment (sakshi006) [inline components/Badge/styleBadge.css:48]: ` top: -4.5rem;
    right: 1.5rem; `

**Review by sakshi006 — CHANGES_REQUESTED** [review]
Added some styling changes.

**Pull request merged**